test_scripts/UpdateVideoStreamingCapabilities/003_3_1_GetSystemCapability_after_OnSystemCapabilityUpdated_other_app.lua

```diff
@@ -0,0 +1,77 @@
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0296-Update-video-streaming-capabilities-during-ignition-cycle.md
+--
+-- Description:  SDL successfully transfer received from HMI on startup videoStreamingCapabilities
+--  to the application after updates of them for other application
+--
+-- Preconditions:
+-- 1. SDL and HMI are started
+-- 2. SDL received videoStreamingCapabilities from HMI
+-- 3. Application 1 is registered and activated
+-- 4. Application 1 is subscribed on OnSystemCapabilityUpdated notification with VIDEO_STREAMING capability type
+--
+-- Sequence:
+-- 1. HMI sends OnSystemCapabilityUpdated notification with updates of VIDEO_STREAMING capability type for Application 1
+-- SDL does:
+-- - a. resend OnSystemCapabilityUpdated notification with updates of VIDEO_STREAMING capability type
+--    to the Application 1
+-- 2. Application 2 is registered and requests videoStreamingCapabilities via GetSystemCapability RPC
+-- SDL does:
+-- - a. send response to the Aplication 2 with received from HMI on startup videoStreamingCapabilities
+--    which stored internally
+-- - b. not request videoStreamingCapabilities from HMI
+---------------------------------------------------------------------------------------------------
+-- [[ Required Shared libraries ]]
+local common = require('test_scripts/UpdateVideoStreamingCapabilities/common')
+
+--[[ Local Variables ]]
+local appSessionId1 = 1
+local appSessionId2 = 2
+local expected = 1
+
+local vsc = common.getVideoStreamingCapability(5)
+vsc.additionalVideoStreamingCapabilities[1].preferredResolution = { resolutionWidth = 1920, resolutionHeight = 1080 }
+vsc.additionalVideoStreamingCapabilities[2].preferredResolution = { resolutionWidth = 1024, resolutionHeight = 768 }
+vsc.additionalVideoStreamingCapabilities[5].preferredResolution = { resolutionWidth = 15, resolutionHeight = 2 }
+
+local function getSystemCapability(pAppId)
```

Comment:
Why do not you use getSystemCapability from `common` file? The comment is related also to other scripts.

Reply:
@GetmanetsIrina Because this function checks the RPC more precisely than the same function from common module. Also the function from common module is used in scrips which does not aimed on testing of GetSystemCapability RPC. That why this detailed function has not moved to common module replacing existing one.